examples/cn0556/README.md

```diff
@@ -0,0 +1,29 @@
+# CN0556
+## 1. Install Prerequisite:
+
+```cmd
+pip install -r requirements.txt
+pip install -e .
```

Comment:
This is already part of the main README instructions.

Reply:
Removed the step 1 - requirements installation step in README file.